pkg/scheduler/internal/cache/cache.go

```diff
@@ -407,7 +407,9 @@ func (cache *schedulerCache) AddPod(pod *v1.Pod) error {
 			// The pod was added to a different node than it was assumed to.
 			klog.Warningf("Pod %v was assumed to be on %v but got added to %v", key, pod.Spec.NodeName, currState.pod.Spec.NodeName)
 			// Clean this up.
-			cache.removePod(currState.pod)
+			if err = cache.removePod(currState.pod); err != nil {
+				return err
```

Comment:
change this to a ```klog.Errorf("removing pod: %v", err)``` instead of returning an error.

Reply:
done, thanks!

Reply:
ping @ahg-g for review